doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_custom_pcb.rst

```diff
@@ -77,8 +64,9 @@ BICR allows for the configuration of various components on your custom board, li
 * Tamper switches
 * Active shield channels
 
-You can find the details of each register contained in BICR in the relevant `BICR register's PDF file`_.
-When not set, the register's default value is ``0xFFFFFFFF``.
+You can find the details in the DTS specification for the BICR in :file:`sdk-zephyr/dts/bindings/misc/nordic,nrf-bicr.yaml`
+
+When not set, the registers' default value is ``0xFFFFFFFF``.
```

Comment:
From above:
"The Board Information Configuration Registers (BICR) are non-volatile memory (NVM) registers"
And here, we talk about a single register. Is this one or more?

Reply:
This is out of scope of this guide. Registers should be part of a separate PDF document.
If it is confusing, I can remove this reference completely @hellesvik-nordic